alerts.go

```diff
@@ -43,26 +44,66 @@ type Alert struct {
 	ClosedAt  int64   `json:"closedAt,omitempty"`
 }
 
-// FindAlerts find alerts
-func (c *Client) FindAlerts() ([]*Alert, error) {
-	req, err := http.NewRequest("GET", c.urlFor("/api/v0/alerts").String(), nil)
+// NextID is next id of alert
+type NextID string
+
+// Data includes alert and next id
+type Data struct {
+	Alerts []*Alert `json:"alerts"`
+	ID     NextID   `json:"nextId,omitempty"`
+}
+
+var d Data
+
+func (c *Client) findAlertsWithParam(v url.Values) (Data, error) {
+	u := c.urlFor("/api/v0/alerts")
+	u.RawQuery = v.Encode()
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s?%s", u.String(), v.Encode()), nil)
 	if err != nil {
-		return nil, err
+		return d, err
```

Comment:
When calling findAlertsWithParam 2 times and only 2nd http request fails, this returns `d`, which contains 1st request's response.
(Moving `var d` into inside `findAlertsWithParam` should solve this, too)